node/forest_libp2p/src/service.rs

```diff
@@ -258,7 +258,7 @@ where
                             &pubsub_block_str,
                             &pubsub_msg_str,).await;
                     },
-                    None => { break; },
+                    None => { break Ok(()); },
```

Comment:
How about moving the `Ok(())` such that you only write it once?

Reply:
Done